docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -0,0 +1,90 @@
+.. ##
+.. ## Copyright (c) 2016-17, Lawrence Livermore National Security, LLC.
+.. ##
+.. ## Produced at the Lawrence Livermore National Laboratory
+.. ##
+.. ## LLNL-CODE-689114
+.. ##
+.. ## All rights reserved.
+.. ##
+.. ## This file is part of RAJA.
+.. ##
+.. ## For details about use and distribution, please read RAJA/LICENSE.
+.. ##
+
+.. _policies-label:
+
+==================
+Execution Policies
+==================
+
+The following serves as reference to the various policies which ``RAJA`` supports.
+
+
+.. note:: * All RAJA execution policies are in the namespace ``RAJA``.
+
+
+--------------------
+Serial/SIMD Policies
+--------------------
+
+* ``seq_exec``  - Enforces sequential loop iterations.
+* ``loop_exec`` - Allows the compiler to vectorize.
+* ``simd_exec`` - Introduces vectorization hints to loop iterations.
+
+---------------
+OpenMP Policies
+---------------
+
+* ``omp_for_exec`` - Distributes loop iterations within threads (static scheduling by default).
+* ``omp_for_nowait_exec`` - Removes synchronization within threaded regions.
+* ``omp_for_static`` - Distributes loop iterations within threads using a static schedule.
+* ``omp_parallel_exec`` - Creates a parallel region.
+* ``omp_parallel_for_exec`` - Creates a parallel region and divides loop iterations between threads.
+* ``omp_parallel_segit`` - Creates a parallel region for index segments.
+* ``omp_parallel_for_segit`` - Create a parallel region for index segments and divide segments between threads.
+* ``omp_collapse_nowait_exec`` - Collapses multiple iteration spaces into a single space and removes any implied barriers.
+
+----------------------
+OpenMP Target Policies
+----------------------
+
+* ``omp_target_parallel_for_exec`` - Variables are mapped to a device environment and a parallel region is created in which loop iterations are divided among threads.
+
+------------
+TBB Policies
+------------
+
+* ``tbb_for_exec`` - Schedules tasks to operate in parallel using the static scheduler.
+* ``tbb_for_static`` - Implements the parallel_for method and uses a static scheduler.
+* ``tbb_for_dynamic`` - Implements the parallel_for method and uses a dynamic scheduler.
+* ``tbb_segit`` - Implements the parallel_for for indexset segments.
+
+-------------
+CUDA Policies
+-------------
+
+Following the CUDA nomenclature, GPU computations are performed on a predefined compute grid.
+Each unit of the grid is referred to as a thread and threads are furthered grouped into
+thread blocks. Threads and thread blocks may have up to three-dimensional indexing for convenience.
+Each CUDA policy requires the user to specify the number of threads in each dimension of a thread block.
+The total number of blocks needed are determined based on the iteration space and the number of threads
+per block. As a starting point, the following policy may be used with the ``RAJA::forall`` loop
+
+* ``cuda_exec<int BlkSz>`` where BlkSz corresponds to the number of threads in a given block.
```

Comment:
I think you should use `BLOCK_SIZE` or something that jumps out a bit more for the policies that have parameters. Same comment for the `*_threadblock_*` policies below.